pupa/scrape/schemas/vote.py

```diff
@@ -7,7 +7,7 @@
     "properties": {
         'identifier': {"type": "string", "blank": True,},
         'motion_text': {"type": "string" },
-        'motion_classification': {"items": {"type": "string", "enum": common.VOTE_CLASSIFICATIONS},
+        'motion_classification': {"items": {"type": "string"},
```

Comment:
If Sunlight needs to ensure `motion_classification` sticks to a few values for its purposes, then it can patch the schemas in its scrapers like I do: https://github.com/opencivicdata/scrapers-ca/blob/master/patch.py

It makes no sense for a generic library to limit motion classification to such a restricted set.